react-components/src/architecture/base/commands/BaseEditTool.ts

```diff
@@ -34,6 +34,10 @@ export abstract class BaseEditTool extends NavigationTool {
     return 'crosshair';
   }
 
+  public override setCursor(value: string): void {
+    this.renderTarget.cursor = value;
+  }
```

Comment:
Why do we need to override it in this class when the implementation is identical to the one in the base class?